tests/core/editable/insertionpreserveid.js

```diff
@@ -1,41 +1,40 @@
 /* bender-tags: editor,unit,insertion */
 
-(function() {
+( function() {
 	'use strict';
 
 	bender.editors = {
 		paragraph: {
-			name: "classic",
+			name: 'classic',
 			config: {
-				allowedContent: "span em u p[id](*){*}",
+				allowedContent: 'span em u p[id](*){*}',
 				enterMode: CKEDITOR.ENTER_P
 			}
 		},
 		divarea: {
-			name: "divarea",
+			name: 'divarea',
 			config: {
-				allowedContent: "span em u div[id](*){*}",
+				allowedContent: 'span em u div[id](*){*}',
 				enterMode: CKEDITOR.ENTER_DIV
 			}
 		}
 	};
 
-	bender.test({
+	bender.test( {
 		'test insert nothing into inline element with id': function() {
 			var editor = this.editors.paragraph,
 				bot = this.editorBots.paragraph;
-			bot.setHtmlWithSelection( '<p style="text-align: right;"><span class="marker" id="abc">te^st</span></p>' );
+			bot.setHtmlWithSelection( '<p style="text-align: right"><span class="marker" id="abc">te^st</span></p>' );
```

Comment:
The `bot.setHtmlWithSelection` uses deprecated `bender.tools.setHtmlWithSelection`, you should use `bender.tools.selection.setWithHtml` instead.